patches/server/0011-Use-TerminalConsoleAppender-for-console-improvements.patch

```diff
@@ -64,7 +65,20 @@ index 220d2696bf40d1657c87d049f563ccfffed2e8ad..c808f36b30d0f8edb8365875039fd20b
      implementation("org.apache.logging.log4j:log4j-iostreams:2.22.1") // Paper - remove exclusion
      implementation("org.ow2.asm:asm-commons:9.7.1")
      implementation("org.spongepowered:configurate-yaml:4.2.0-SNAPSHOT") // Paper - config files
-@@ -92,6 +112,19 @@ tasks.check {
+@@ -71,6 +92,12 @@ tasks.jar {
+     }
+ }
+ 
++// Paper start - allow native access for jline-ffm
++rootProject.tasks.named<io.papermc.paperweight.tasks.CreateBundlerJar>("createMojmapBundlerJar") {
```

Comment:
Kind of annoying that we have to resort to cross-project configuration here, since putting it in the root build file would require forks to copy the diff and it wouldn't be associated with the patch. Maybe we should just add the attribute by default in paperweight; it would still allow adding extras manually and removing this one if needed.